search-app/docs/api.md

```diff
@@ -3140,6 +3137,65 @@ Content-Length: 292
 }
 ```
 
+UMM JSON response:
+
+```
+curl -g -i "%CMR-ENDPOINT%/variables.umm_json?variable_name=Variable1234&pretty=true"
+HTTP/1.1 200 OK
+Content-Type: application/vnd.nasa.cmr.umm_results+json;version=1.9; charset=UTF-8
```

Comment:
From a documentation stand point this seems misleading. Versioning isnt implemented yet and I don't recognize this Content-Type, is it accurate?